src/App.tsx

```diff
@@ -1,5 +1,13 @@
+import { ReactLocation, Router } from '@tanstack/react-location';
+
```

Comment:
```suggestion

const location = new ReactLocation();
```

I think it's a better idea if we put this outside of app 
what you think?